lib/codeviewer/code_segments.dart

```diff
@@ -44312,37 +44312,30 @@ class CodeSegments {
       TextSpan(style: codeStyle.keywordStyle, text: 'import'),
       TextSpan(style: codeStyle.baseStyle, text: ' '),
       TextSpan(
-          style: codeStyle.stringStyle, text: '\u0027dart:collection\u0027'),
-      TextSpan(style: codeStyle.baseStyle, text: ' show '),
-      TextSpan(style: codeStyle.classStyle, text: 'IterableMixin'),
-      TextSpan(style: codeStyle.punctuationStyle, text: ';'),
-      TextSpan(style: codeStyle.baseStyle, text: '\u000a'),
-      TextSpan(style: codeStyle.keywordStyle, text: 'import'),
-      TextSpan(style: codeStyle.baseStyle, text: ' '),
-      TextSpan(style: codeStyle.stringStyle, text: '\u0027dart:math\u0027'),
+          style: codeStyle.stringStyle,
+          text: '\u0027package:flutter/material.dart\u0027'),
       TextSpan(style: codeStyle.punctuationStyle, text: ';'),
       TextSpan(style: codeStyle.baseStyle, text: '\u000a'),
       TextSpan(style: codeStyle.keywordStyle, text: 'import'),
       TextSpan(style: codeStyle.baseStyle, text: ' '),
-      TextSpan(style: codeStyle.stringStyle, text: '\u0027dart:ui\u0027'),
-      TextSpan(style: codeStyle.baseStyle, text: ' show '),
-      TextSpan(style: codeStyle.classStyle, text: 'Vertices'),
+      TextSpan(
```

Comment:
These changes are being generated in a version before this change as well (after updating code-segments) : 

```dart
    TextSpan(
          style: codeStyle.stringStyle,
          text: '\u0027package:flutter/material.dart\u0027'),
      TextSpan(style: codeStyle.punctuationStyle, text: ';'),
      TextSpan(style: codeStyle.baseStyle, text: '\u000a'),
      TextSpan(style: codeStyle.keywordStyle, text: 'import'),
      TextSpan(style: codeStyle.baseStyle, text: ' '),
      TextSpan(
          style: codeStyle.stringStyle,
          text:
              '\u0027package:flutter_gen/gen_l10n/gallery_localizations.dart\u0027'),
      TextSpan(style: codeStyle.punctuationStyle, text: ';'),
      TextSpan(style: codeStyle.baseStyle, text: '\u000a\u000a'),
```

which should not be there IMO. 

Imports in hosted version : 
![before](https://user-images.githubusercontent.com/33422449/154854855-2ab6ce7e-3ba4-429a-b048-0cd9e4f3540c.png)

Imports after this change :
![after](https://user-images.githubusercontent.com/33422449/154854915-4556b84f-2754-4bea-860c-3aae5887569b.png)

Reply:
I can't find the issue, but for some unknown reason, the code viewer generates differently on non-Unix platforms. Can you revert the changes to the the code-viewer?

Reply:
yeah, will revert the code-segments changes.

Reply:
@guidezpl can you review and add these.